src/lib/openapi/spec/admin-features-query-schema.test.ts

```diff
@@ -11,7 +11,7 @@ test('adminFeaturesQuerySchema empty', () => {
 
 test('adminFeatureQuerySchema all fields', () => {
     const data: AdminFeaturesQuerySchema = {
-        tag: [['some-tag', 'some-other-tag']],
+        tag: ['some-tag', 'some-other-tag'],
```

Comment:
If tags need to be `type:value`, then this test should fail because these tags are not.